ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2845,6 +2845,18 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-sdn
+- as: vsphere-ipi-ovn-migration-sdn-p2-f7
```

Comment:
Would you mind to sort the tests based on the name, also add it to 4.14, too

Reply:
@liangxia I though I insert file based on the names as below? Please advise, thanks! 
- as: vsphere-ipi-sdn-p2-f7
- as: vsphere-ipi-ovn-migration-sdn-p2-f7
- as: vsphere-ipi-zones-p2-f7

Reply:
IMO, vsphere-ipi-ovn should sort before vsphere-ipi-sdn